.eslintrc.js

```diff
@@ -178,6 +178,7 @@ module.exports = {
         'aws-toolkits/no-console-log': 'error',
         'aws-toolkits/no-json-stringify-in-log': 'error',
         'aws-toolkits/no-printf-mismatch': 'error',
+        'aws-toolkits/no-foreach': 'error',
```

Comment:
We depend on `eslint-plugin-unicorn` so I think we can use this rule: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-array-for-each.md

Reply:
Good idea! That rule also comes with a way to auto-fix _most_ cases so I switched over to it completely here: https://github.com/aws/aws-toolkit-vscode/pull/6281